packages/core/src/synthesis/Synthesizer.ts

```diff
@@ -695,10 +695,8 @@ export class Synthesizer {
         possibleOps = del ? [del] : [];
       }
     }
-    if (possibleOps) {
-      log.debug(`Found mutations for delete:\n${showMutations(possibleOps)}`);
-      return possibleOps;
-    } else return [];
+    log.debug(`Found mutations for delete:\n${showMutations(possibleOps)}`);
```

Comment:
This change makes this part of the routine much simpler -- thanks for doing that!  Question: should we continue to wrap the debug message in a conditional to only log "mutations found" when possibleOps is not [] -- I guess it doesn't hurt anything to log it in either case?

Reply:
Sure, I can make that change! I'm guessing that was @wodeni's original intent?